Split/Network/Streaming/PushManagerEventBroadcaster.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  PushManagerEventBroadcaster.swift
+//  SplitTests
+//
+//  Created by Javier L. Avrudsky on 26/08/2020.
+//  Copyright © 2020 Split. All rights reserved.
+//
+
+import Foundation
+
+enum PushStatusEvent {
+    case disablePolling
+    case enablePolling
+}
+
+protocol PushManagerEventBroadcaster {
+    typealias IncomingMessageHandler = (PushStatusEvent) -> Void
+    func push(event: PushStatusEvent)
+    func register(handler: @escaping IncomingMessageHandler)
+    func stop()
```

Comment:
what about renaming this method to `deregisterAll`? (`stop` makes me think that the protocol should also have a `start` method).